src/index.js

```diff
@@ -32,10 +32,16 @@ export {default as Layer} from './core/lib/layer';
 export {default as CompositeLayer} from './core/lib/composite-layer';
 
 // Viewports
-export {default as Viewport} from './core/viewports/viewport';
-export {default as FirstPersonViewport} from './core/viewports/first-person-viewport';
-export {default as ThirdPersonViewport} from './core/viewports/third-person-viewport';
-export {default as WebMercatorViewport} from './core/viewports/web-mercator-viewport';
+export {
+  Viewport,
+  FirstPersonViewport,
+  ThirdPersonViewport,
+  WebMercatorViewport,
+  PerspectiveViewport,
+  OrthographicViewport,
+  // TODO: orbit-viewport to be merged w/ third-person-viewport
+  OrbitViewport
+} from './core';
```

Comment:
Seems we are exporting using explicit path in both /index.js and /core/index.js.
I can do around of cleanup if no objection.